### PR TITLE
Newsletters: add 214 (2022-08-24)

### PR DESCRIPTION
**Pull request by harding**
- [x] Releases/RCs & topic entries @harding 
- [x] Client/Services section @bitschmidty 
- [x] Bitcoin Core 25504 @Xekyo

### CODE REVIEW

File: _posts/en/newsletters/2022-08-24-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+
+- [BDK 0.21.0][] is the latest release of this library for building
+  wallets.
+
+- [Core Lightning 0.12.0rc2][] is a release candidate for the next major
+  version of this popular LN node implementation.
+
+- [LND 0.15.1-beta.rc1][] is a release candidate that "includes support
+  for [zero conf channels][topic zero-conf channels], scid [aliases][],
+  [and] switches to using [taproot][topic taproot] addresses everywhere
```
Comment (glozow) [inline _posts/en/newsletters/2022-08-24-newsletter.md:82]: Unless I'm missing something?
```suggestion
  and switches to using [taproot][topic taproot] addresses everywhere
```

File: _posts/en/newsletters/2022-08-24-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+  applicable by default".
```
Comment (glozow) [inline _posts/en/newsletters/2022-08-24-newsletter.md:83]: I have a hard time parsing "using something applicable by default"

```suggestion
  by default".
```

File: _posts/en/newsletters/2022-08-24-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+*Notable changes this week in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #25504][] amends the responses to `listsinceblock`,
+  `listtransactions`, and `gettransactions` to state in a new field
+  `parent_descs` the associated descriptors. Additionally,
```
Comment (bitschmidty) [inline _posts/en/newsletters/2022-08-24-newsletter.md:97]: perhaps ```suggestion
  ,`parent_descs`, the associated descriptors. Additionally,
```

File: _posts/en/newsletters/2022-08-24-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+  outputs are omitted as implicit by-products of outbound payments, but
+  listing them may be interesting in the context of watch-only
+  descriptors.
+
+- [Eclair #2234][] adds support for associating a DNS name with a node
+  in its announcements as now allowed by [BOLTs #911][] (see [Newsletter
+  #212][news212 bolts911]).
+
+- [LDK #1503][] adds support for [onion messages][topic onion messages]
+  as defined by [BOLTs #759][].  The PR indicates that is in
```
Comment (bitschmidty) [inline _posts/en/newsletters/2022-08-24-newsletter.md:109]: perhaps 
```suggestion
  as defined by [BOLTs #759][].  The PR indicates that this change is in
```

File: _posts/en/newsletters/2022-08-24-newsletter.md [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+
+- [LND #6596][] adds a new `wallet addresses list` RPC that lists all of the
+  wallet's addresses and their current balance.
+
+- [BOLTs #1004][] begins recommending that nodes which maintain
+  information about channels for routing should wait at least 12 blocks
+  after a channel is closed before deleting their information it.  This
+  delay will support detection of [splices][topic splicing] where a
+  channel isn't actually closed but instead has funds added or removed
+  to it in an onchain transaction.
```
Comment (bitschmidty) [inline _posts/en/newsletters/2022-08-24-newsletter.md:120]: added or removed "from" it?

File: _topics/en/silent-payments.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+
+## Optional.  Force the display (true) or non-display (false) of stub
+## topic notice.  Default is to display if the page.content is below a
+## threshold word count
+#stub: false
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+## Should be less than 500 characters
+excerpt: >
+  **Silent payments** are a type of payments that can be made to a
```
Comment (bitschmidty) [inline _topics/en/silent-payments.md:48]:
```suggestion
  **Silent payments** are a type of payment that can be made to a
```

File: _posts/en/newsletters/2022-08-24-newsletter.md (hunk 1 above)
Comment (harding) [reply _posts/en/newsletters/2022-08-24-newsletter.md:83]: This is a quote from their release notes where elided some details and inserted the conjunction.  I put the modification in brackets to indicate it's not original (i.e., this is deliberate and not releated to markdown).

**Comment by harding** [comment]
Made suggested edits (thanks @glozow and @bitschmidty !), review other contributions (thanks @Xekyo and @bitschmidty !), updated releases/RCs, and added topic links.  Thanks everyone!

**Comment by bitschmidty** [comment]
Looks good, thank you @harding @Xekyo and @glozow for your contributions this week!

**Pull request merged**